frontend/src/metabase/query_builder/components/expressions/ExpressionEditorHelpText.tsx

```diff
@@ -22,48 +22,55 @@ type HelpText = {
 interface HelpTextProps {
   helpText: HelpText;
   width: number;
+  target: Element;
 }
 
-const HelpText = ({ helpText, width }: HelpTextProps) =>
-  helpText ? (
-    <Popover
-      tetherOptions={{
-        attachment: "top left",
-        targetAttachment: "bottom left",
-      }}
-      style={{ width }}
-      isOpen
-    >
-      {/* Prevent stealing focus from input box causing the help text to be closed (metabase#17548) */}
-      <div onMouseDown={e => e.preventDefault()}>
-        <p
-          className="p2 m0 text-monospace text-bold"
-          style={{ background: colors["bg-yellow"] }}
-        >
-          {helpText.structure}
-        </p>
-        <div className="p2 border-top">
-          <p className="mt0 text-bold">{helpText.description}</p>
-          <p className="text-code m0 text-body">{helpText.example}</p>
-        </div>
-        <div className="p2 border-top">
-          {helpText.args.map(({ name, description }, index) => (
-            <div key={index}>
-              <h4 className="text-medium">{name}</h4>
-              <p className="mt1 text-bold">{description}</p>
+const HelpText = ({ helpText, width, target }: HelpTextProps) => {
+  if (!helpText) {
+    return null;
+  }
+
+  return (
+    <TippyPopover
+      maxWidth={width}
+      reference={target}
```

Comment:
Since the previous `Popover` relies on an implicit parent container, we need to use `reference` and pass the parent as a ref.

This is similar to the migration in https://github.com/metabase/metabase/pull/21545/